scripts/biomedical/humanMetabolomeDatabase/README.md

```diff
@@ -0,0 +1,127 @@
+
+### Schema Overview
+
+The schema representing metabolome and proteome data from HMDB is defined in [HMDB.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list.mcf) and [HMDB_enum.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list_enum.mcf). The tmcfs for each of the corresponding csv files can be found [here](https://github.com/suhana13/data/tree/add_hmdb_metabolites/scripts/biomedical/humanMetabolomeDatabase/tmcf).
+
+The Metabolite entity is used to describe chemical substances which are intermediates or end products in a metabolic reaction. It has several text value properties, representing "humanMetabolomeDatabaseID", "monoIsotropicWeight", "iupacName", "chemicalName", "chemicalFormula", "inChIKey", "casRegistry", "simplifiedMolecularInputLineEntrySystem", "drugBankID", "chebiID", "pubChemCompoundID", "phenolExplorerCompoundID", "foodB", "knapsack", "chemSpiderID", "keggCompoundID", "metaCycID", "biggID", "metlinID", "proteinDataBankID", "drugLogP", and various enumeration classes, namely, "ChemicalCompoundCategoryEnum", "ChemicalCompoundParentEnum", "ChemicalCompoundSuperClassEnum", "ChemicalCompoundClassEnum", "ChemicalCompoundSubClassEnum", and "ChemicalCompoundMolecularFrameworkEnum".
+
+The Protein entity is used to describe a class of nitrogenous organic compounds that are composed of large molecules called amino acids. It has several text value properties representing, "uniProtID", "genBankID", "geneCardID", "hgncID", "proteinGeneralFunction", "proteinSpecificFunction", "aminoAcidResidueNumber", and various enumeration classes, namely, "KeggProteinPathway", and "CellularCompartment".
+
+- ### Notes and Caveats
+
+The ChEMBL Id for the metabolites was missing, so it had to be generated using other databases, like VMH and HumanGEM-Human1, and also using the bioservices package. In addition, the proteins in hmdb were mapped to preexisting protein entities in Data Commons knowledge graph by querying multiple properties to identify the corresponding dcid. The dcid was manually currated in the handful of cases in which no matches were programatically found. Also, all the original data was formatted in xml format necesitating parsing which contributed to the total increase in runtime of the wrapper bash script.
+
+- ### License
+
+This data is freely available to all users. However, the commercial usage of the database requires explicit permission of the authors. More information on the HMDB license can be found [here](https://pubchem.ncbi.nlm.nih.gov/source/811).
+
+## About the import
+
+- ### Artifacts
+
+- #### Scripts
+
+`hmdb_metabolite_xml_csv.py`
+`format_metabolite.py`
+`hmdb_metabolite_add_chembl.py`
+`format_hmdb_metabolites.py`
+`hmdb_protein_xml_csv.py`
+`format_hmdb_protein.py`
+`format_hmdb_protein_metabolite.py`
+`format_hmdb_protein_pathway.py`
+`parse_hmdb_go.py`
+`format_hmdb_go.py`
+
+## Examples
+
```

Comment:
Files needed to run this script including CHEMBL_HMDB_map.csv should be stored in this directory and listed as artifacts with hyperlinks

Reply:
@spiekos, we are generating `CHEMBL_HMDB_map.csv`, in the steps preceding this, would still want it to be added to the directory ?